### DISCUSSION TITLE
Moving splits

### DISCUSSION THREAD

**Discussion by EmreDogann** [JetBrains/ideavim#696] [General]
In Vim, to move existing splits around, you can do the following:
```
<C-W>H		" Move split left
<C-W>J		" Move split down
<C-W>K		" Move split up
<C-W>L		" Move split right
```

However, these mappings do not work in ideavim. Do you know if these mappings are possible, or if not are there any alternatives?
Also, as far as I'm aware there is no way of creating splits to the left/above the current split correct?

**Comment by citizenmatt** [comment]
Unfortunately, the move commands aren't currently supported: [VIM-2736](https://youtrack.jetbrains.com/issue/VIM-2736/Support-Ctrl-W-uppercase-HJKL-to-move-window).
And there currently isn't a way to create splits to the left or above: [VIM-1670](https://youtrack.jetbrains.com/issue/VIM-1670/Support-splitright-and-splitbelow)
